src/app/plan/components/Wizard/HooksFormContainer.tsx

```diff
@@ -0,0 +1,87 @@
+import { withFormik, FormikProps } from 'formik';
+import utils from '../../../common/duck/utils';
+import HooksFormComponent from './HooksFormComponent';
+
+const AddEditHooksFormContainer: any = withFormik({
+  mapPropsToValues: ({ initialHookValues }) => {
+    const values = {
+      hookName: '',
+      hookImageType: 'ansible',
+      customContainerImage: 'quay.io/konveyor/hook-runner:latest',
+      ansibleRuntimeImage: 'quay.io/konveyor/hook-runner:latest',
+      clusterType: 'source',
+      srcServiceAccountName: '',
+      destServiceAccountName: '',
+      srcServiceAccountNamespace: '',
+      destServiceAccountNamespace: '',
+      migrationStep: '',
+      ansibleFile: ''
+    };
```

Comment:
We should also maybe add an annotation here, `const values = { ... }: IFormValues;` so that TS can help us with missing fields / bad types here. We should probably do the same in the WizardContainer tbh. There were some object-type fields that were initialized with empty arrays there that I found in my earlier work.